src/dto/rata-traverse.dto.ts

```diff
@@ -233,19 +233,33 @@ export class RataTraverseBaseDTO {
   numberWallEffectsPoints?: number;
 
   @IsOptional()
-  @IsInt()
-  @IsInRange(-99, 99, {
+  @IsNumber(
+    { maxDecimalPlaces: 1 },
+    {
+      message: (args: ValidationArguments) => {
+        return `The value of [${args.value}] for [${args.property}] is allowed only 1 decimal place for [${KEY}]`;
+      },
+    },
+  )
+  @IsInRange(-99999.9, 90, {
     message: (args: ValidationArguments) => {
-      return `The value of [${args.value}] for [${args.property}] must be within the range of -99 and 99 for [${KEY}]`;
+      return `The value of [${args.value}] for [${args.property}] must be within the range of -99999.9 and 90 for [${KEY}]`;
     },

```

Comment:
Double check these min and max values.  How can an angle be greater or less than 360/-360 ?  And should the min and max be negative/positive of the same value?

Reply:
Let me check with @jwhitehead77 . 

This is what in the audit sheet.
![image](https://github.com/US-EPA-CAMD/easey-qa-certification-api/assets/102534033/c580e66f-aab4-489a-8d12-2059ac53412d)

Reply:
updated.